mods/ra/rules/vehicles.yaml

```diff
@@ -24,14 +24,24 @@ V2RL:
 		Range: 4c0
 	Armament:
 		Weapon: SCUD
+		PauseOnCondition: !ammo
+	AmmoPool:
+		Ammo: 1
+		PipCount: 0
+		AmmoCondition: ammo
+	ReloadAmmoPool:
+		Delay: 240
+		Count: 1
 	AttackFrontal:
+	WithFacingSpriteBody@LOADED:
+		RequiresCondition: ammo
+	WithFacingSpriteBody@EMPTY:
```

Comment:
This still isn't adding a `-WithFacingSpriteBody:` nor removed the `@LOADED`. That means the game currently renders an additional loaded sprite all the time (although the empty sprite may be displayed on top of it so it isn't immediately noticable).

Reply:
Good catch.  Removing the `@LOADED` would be the better choice.